dev-client/src/screens/SettingsScreen/components/UserIndicatorComponent.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useTranslation} from 'react-i18next';
+import {Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {useSelector} from 'terraso-mobile-client/store';
+
+export function UserIndicator() {
+  const currentUser = useSelector(state => state.account.currentUser.data);
+  const {t} = useTranslation();
+
+  return (
+    <Text variant="body1">
+      {currentUser?.email ?? t('settings.unknownUser')}
```

Comment:
This shouldn't ever happen in practice, but the `currentUser` selector can possibly return an empty value so this seemed good for completion's sake.